contracts/parent-chain/contract-upgrades/EspressoOspMigrationAction.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.8.9;
+
+import "forge-std/Script.sol";
+import "@arbitrum/nitro-contracts/src/challenge/IChallengeManager.sol";
+import "@arbitrum/nitro-contracts/src/rollup/IRollupCore.sol";
+import "@arbitrum/nitro-contracts/src/rollup/IRollupAdmin.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+error IncorrectWasmModuleRoot(bytes32 incorrectAddr);
```

Comment:
The errors should be a part of the contract, so I think you should add them inside the contract object